README.md

```diff
@@ -80,10 +74,10 @@ where `pos` is the unique ID representing this inserted character's location,
 `chr` is the inserted character, and `prev` is either the preceding character's
 ID, or `null` (this character is a document root).
 
-### str.delete(pos, cb)
+### str.delete(pos, [count], [cb])
```

Comment:
I'm really leery of spectral optional parameters, especially when they're in the middle and the code needs to start transposing args around to work. We're making a breaking API change anyways, so let's make `count` mandatory.